bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/vacuumThing.xml

```diff
@@ -106,11 +108,16 @@
 		<item-type>Switch</item-type>
 		<label>Do Not Disturb</label>
 	</channel-type>
-	<channel-type id="error_code">
+	<channel-type id="error">
```

Comment:
When has the `vacuumThing` been introduced? If it is already present in 2.5.0 this is a breaking change.  In any case, documentations needs to be updated.

Reply:
That's right, it should be avoided. I'm going to rename it back as originally and give the new channel a different name, even though the name `error_code` for the String Channel was chosen a little unfortunately in the past.